filters/include/pcl/filters/convolution.h

```diff
@@ -216,9 +226,9 @@ namespace pcl
         int half_width_{};
         /// kernel size - 1
         int kernel_width_{};
+        /// number of threads to use
+        unsigned int num_threads_{0};
```

Comment:
Should probably be left at 1 or set via `setNumberOfThreads` in the constructor?